charts/posthog/templates/_snippet-plugins-deployment.tpl

```diff
@@ -96,6 +100,11 @@ spec:
         # Expose the port on which the healtchheck endpoint listens
         - containerPort: 6738
         env:
+        {{ if .params.volumeMounts }}
```

Comment:
So much abstraction/templating! I was the person that added the templating here, I feel like eventually we should just have separate definitions, but that’s ot